app/controllers/oidc/concerns/trusted_publisher_controller.rb

```diff
@@ -0,0 +1,61 @@
+module OIDC::Concerns::TrustedPublisherController
```

Comment:
This name is confusing. Maybe TrustedPublisherCreation or Actions. Anything except Controller.